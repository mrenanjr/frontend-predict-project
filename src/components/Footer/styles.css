.footer-container {
    position: fixed;
    left: 0;
    bottom: 0;
    height: 60px;
    width: 100%;

    display: flex;
    justify-content: center;
    align-items: center;

    background: linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, #0067AC 100%), linear-gradient(180deg, rgba(0, 103, 172, 0.1) 0%, rgba(255, 255, 255, 0.1) 100%);
}

.footer-container .img {
    height: 60px;
    width: 150px;
    justify-self: end;

    background: #C0C0C0;

    margin-right: 60px;
}

.footer-container .img img {
    margin: auto;
    display: block;
    height: 100%;
}

.footer-copyright {
    flex: 1;
    justify-content: center;
    display: flex;

    margin-left: 210px;

    color: white;
}