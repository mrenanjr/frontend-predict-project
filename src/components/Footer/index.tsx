import React from 'react';
import './styles.css';

import Img from '../../assets/insert_photo-24px.svg';

const Footer = () => {
    return (
        <footer className="footer-container">
            <div className="footer-copyright">
                Â©2020 Copyright
            </div>
            <div className="img">
                <img src={Img} alt="Imagem"/>
            </div>
        </footer>
    );
}

export default Footer;
