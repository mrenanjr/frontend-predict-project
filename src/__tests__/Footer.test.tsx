import * as React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/Footer';

test('Footer loads corretly', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
});