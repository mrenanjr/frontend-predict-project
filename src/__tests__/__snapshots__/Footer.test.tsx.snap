// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer loads corretly 1`] = `
<footer
  className="footer-container"
>
  <div
    className="footer-copyright"
  >
    Â©2020 Copyright
  </div>
  <div
    className="img"
  >
    <img
      alt="Imagem"
      src="test-file-stub"
    />
  </div>
</footer>
`;
